[GEOS-12154] Improved URL validation in SLD processing. (#9712)
geoserver/geoserver: doc/en/user/security/urlchecks.md

- WMS GetMap, GetFeatureInfo and GetLegendGraphic requests with remote SLD stylesheets (`sld` parameter)
- Remote icons referenced by styles (access to icons in the data directory is always allowed)

> - WMS GetMap and GetFeatureInfo requests in feature portrayal mode, either via the `REMOTE_OWS_URL`/`REMOTE_OWS_TYPE` parameters or a `<RemoteOWS>`/`<OnlineResource>` element embedded in an SLD document (`sld` or `sld_body` parameter)

Reviewer: Making a small documentation change to force a new workflow run:
```suggestion
- WMS GetMap and GetFeatureInfo request feature portrayal using mode using `REMOTE_OWS_URL`/`REMOTE_OWS_TYPE` parameters.
- WMS GetMap and GetFeatureInfo request feature portrayal using mode using  `<RemoteOWS>`/`<OnlineResource>` reference in in an SLD document `sld` or `sld_body` element.
```
